### PR TITLE
📊 AI index report - robots

### PR DESCRIPTION
**Pull request by veronikasamborska1994**
hi @paarriagadap! sorry for another review! I am going to have another couple of these but the code is mostly extremely straightforward and rarely changes in logic. no rush to review!

### CODE REVIEW

File: dag/artificial_intelligence.yml [hunk 1]
```diff
@@ -326,4 +275,18 @@ steps:
...
\ No newline at end of file
+    - data://garden/artificial_intelligence/2024-06-28/ai_investment
+
+  # AI Index Report 2024 - AI robots
+  data://garden/artificial_intelligence/2024-06-28/ai_robots:
+     - snapshot://artificial_intelligence/2024-06-28/ai_robots.csv
+  data://grapher/artificial_intelligence/2024-06-28/ai_robots:
+    - data://garden/artificial_intelligence/2024-06-28/ai_robots
+
+  # AI Robots
```
Comment (paarriagadap) [inline dag/artificial_intelligence.yml:286]: You can archive this if no longer needed

**Review by paarriagadap — APPROVED** [review]
All good! (I was waiting for a code to finish). Shame that the data on industrial robots data does not have multiple years. It would be nice to have that.

**Comment by veronikasamborska1994** [comment]
> All good! (I was waiting for a code to finish). Shame that the data on industrial robots data does not have multiple years. It would be nice to have that.

actually they have this data over time for top 5 countries and I agree it's a bit more interesting! I've replaced it now.

**Comment by veronikasamborska1994** [comment]
thank you @paarriagadap :)

**Pull request merged**